scripts/evergreen/e2e/single_e2e.sh

```diff
@@ -48,6 +48,16 @@ deploy_test_app() {
     BUILD_ID="${BUILD_ID:-default_build_id}"
     BUILD_VARIANT="${BUILD_VARIANT:-default_build_variant}"
 
+    if ! chart_info=$(scripts/dev/run_python.sh scripts/release/oci_chart_info.py --build-scenario "${BUILD_SCENARIO}" 2>&1); then
```

Comment:
if we have all of this executed on the host already, why to propagate it to the test pod at all? Couldn't we decide here which helm path to use and use a single HELM_CHART_PATH env var for that that we will use in tests?